modules/dists/CyclicDist.chpl

```diff
@@ -97,7 +97,7 @@ to the ID of the locale to which it is mapped.
     use CyclicDist;
 
     const Dist = new cyclicDist(startIdx=(1,1));
-    const D = {1..8, 1..8} dmapped Dist;
+    const D = cyclicDist.createDomain({1..8, 1..8});
```

Comment:
I've just come across this typo in the doc, should be `const D = Dist.createDomain({1..8, 1..8})`, right?

Reply:
Good catch, thanks! I'll fix it.